### ISSUE TITLE
Text strings longer than 1024 characters are not pre-filled in search dialog

### ISSUE THREAD

**Issue by terabyte-gc** [notepad-plus-plus/notepad-plus-plus#13677]
### Description of the Issue
Selecting a string of text longer than 1024 characters doesn't pre-fill anymore the CTRL+F search dialog after upgrading from 8.4.9 to 8.5.3.

### Steps to Reproduce the Issue
1. Copy this text in a new file:
`this_is_a_long_repeated_string_that_exceeds_1204_characters___this_is_a_long_repeated_string_that_exceeds_1204_characters___this_is_a_long_repeated_string_that_exceeds_1204_characters___this_is_a_long_repeated_string_that_exceeds_1204_characters___this_is_a_long_repeated_string_that_exceeds_1204_characters___this_is_a_long_repeated_string_that_exceeds_1204_characters___this_is_a_long_repeated_string_that_exceeds_1204_characters___this_is_a_long_repeated_string_that_exceeds_1204_characters___this_is_a_long_repeated_string_that_exceeds_1204_characters___this_is_a_long_repeated_string_that_exceeds_1204_characters___this_is_a_long_repeated_string_that_exceeds_1204_characters___this_is_a_long_repeated_string_that_exceeds_1204_characters___this_is_a_long_repeated_string_that_exceeds_1204_characters___this_is_a_long_repeated_string_that_exceeds_1204_characters___this_is_a_long_repeated_string_that_exceeds_1204_characters___this_is_a_long_repeated_string_that_exceeds_1204_characters___this_last_part_now_is_at_1032_characters`
2. Select all the text and press CTRL+F to open the search dialog.

### Expected Behavior
The search field in the search dialog should be **pre-filled with the selected text**.

### Actual Behavior
The search field in the search dialog that opens is **empty**.
I tested also the previous 8.5.x versions, and it seems the issue started with 8.5.0.

### Debug Information
Notepad++ v8.5.3   (64-bit)
Build time : May 15 2023 - 06:09:36
Path : C:\Program Files\Notepad++\notepad++.exe
Command Line : "C:\Users\XXX\Desktop\SEARCH BUG EXAMPLE.txt"
Admin mode : OFF
Local Conf mode : OFF
Cloud Config : OFF
OS Name : Windows 10 Pro (64-bit)
OS Version : 22H2
OS Build : 19045.2846
Current ANSI codepage : 1252
Plugins : 
    HexEditor (0.9.12)
    JsonTools (4.8.1)
    mimeTools (2.9)
    NppConverter (4.5)
    NppExport (0.4)

**Comment by terabyte-gc** [comment]
This is a really annoying issue considering I search long strings multiple times (50+) daily. I reverted back to 8.4.9 for now. 🙄

**Comment by terabyte-gc** [comment]
I see 8.5.4 is out but the fix for this one didn't make it in. Guess I'll keep staying on 8.4.9. 👀

**Comment by terabyte-gc** [comment]
It's been another 2 months, but not even a reply.

I'd like to upgrade to the 8.5.x... Any ETA for this one? 🙄

**Comment by alankilborn** [comment]
@terabyte-gc 

I believe that an upcoming implementation to solve #14108 will allow prefilling the _Find what_ box up to 2046 characters, but no more than that (unsure if 2046 meets your typical use case or not).

**Comment by terabyte-gc** [comment]
After a quick check, the longest (regex) strings I have are 2041 and 2039. It barely fits. 😋

**Comment by terabyte-gc** [comment]
Unfortunately, it doesn't seem this was implemented up to 2046 characters. I just installed 8.5.8 and the field only allows a maximum value of 1024, which is useless for me. I just want the text I select to pre-fill the search field. I don't care about the "In Selected" checkbox. Using the button "Search in all open documents" that setting is ignored anyway. 🤷‍♂️


Back to 8.4.9 again. It doesn't look like I'll be upgrading anytime soon. 😔
